e2e-tests/oauth-flow/openid4vp/run-test.sh

```diff
@@ -53,7 +53,7 @@ echo "---------------------------------------"
 echo "Request access token call"
 echo "---------------------------------------"
 # Request access token
-REQUEST="{\"verifier\":\"${PARTY_A_DID}\",\"scope\":\"test\", \"user_id\":\"1\", \"redirect_uri\":\"http://callback\"}"
+REQUEST="{'verifier':'${PARTY_A_DID}','scope':'test', 'preauthorized_user':{'id':'1', 'name': 'John Doe', 'role': 'Janitor'}, 'redirect_uri':'http://callback'}"
```

Comment:
reversing `"` and `'` works

however, an empty `preauthorized_user` does not fail:
`REQUEST='{"verifier":"${PARTY_A_DID}", "scope":"test", "preauthorized_user":{"id":""}, "redirect_uri":"http://callback"}'`

I had expected the strict handler to catch this since all fields are required, but apparently this is not checked.

Reply:
Added checks and test in the API for this